src/tests/ftest/util/soak_test_base.py

```diff
@@ -92,7 +92,8 @@ def setUp(self):
         self.soak_dir = self.base_test_dir + "/soak"
         self.soaktest_dir = self.soak_dir + "/pass" + str(self.loop)
         # Create the a shared directory for logs
-        self.sharedsoak_dir = self.tmp + "/soak"
+        shared_dir = os.getenv('DAOS_TEST_SHARED_DIR', os.path.expanduser('~/daos_test'))
+        self.sharedsoak_dir = shared_dir + "/soak"
```

Comment:
`self.tmp` should only be an issue when running out of a source build. If running out of `/usr` the problem should not exist, and if it does then there is another problem. I think we should understand what that other problem is before putting a bandaid here

Reply:
I'll also put my `self.tmp` fix in a separate PR so we can get it in